package-lock.json

```diff
@@ -6116,7 +6116,7 @@
           }
         },
         "prettier": {
-          "version": "npm:prettier@1.19.1",
+          "version": "npm:wp-prettier@1.19.1",
```

Comment:
Eh? Will revert this.

Reply:
`@wordpress/scripts`/`@wordpress/eslint-plugin` does some weird aliasing for prettier unfortunately :/

And Dependabot doesn't like that either, see #323.